### PR TITLE
Fix MUSIC example

### PR DESCRIPTION
**Pull request by steffengraber**
This PR fixes the music example for use with NEST 2.18.0 and higher.
The documentation has also been improved.

### CODE REVIEW

**Comment by steffengraber** [comment]
As discussed in the Open NEST Developer Meeting on 28.10.2019, I renewed the terminal output of NEST and MUSIC in the documentation.  I have created this in a docker container. It would be nice if someone could verify that.

File: doc/guides/using_nest_with_music.rst [hunk 1]
```diff
@@ -155,27 +155,53 @@ first NEST process and show the same membrane potential trace.
...
+	2	9	-62.1583
+	2	1	-70
+	2	2	-70
+	2	3	-68.1559
+	2	4	-61.9174
+	2	5	-70
+	2	6	-70
+	2	7	-70
+	2	8	-65.2054
+	2	9	-62.1583
```
[the remark this answers is not in the document]
Comment (steffengraber) [reply doc/guides/using_nest_with_music.rst]: @jougs Is there a way to suppress or shorten the welcome message in python? When I use `nest.set_verbosity("M_QUIET")` in the scripts, the output does not change.

**Comment by jougs** [comment]
I was afraid you're going to come back and ask ;-)

The problem with `setverbosity()` is that it only takes effect _after_ the welcome message has been printed already. The way to suppress _any_ output upon startup (other than those stupid RNG messages, that is; see #700, killed in #1071) goes like this (first code block being shell, second Python)

**In NEST 2.x**
```bash
export DELAY_PYNEST_INIT=1
```
```python
import nest
nest.ll_api.init(["nest", "--quiet"])
```

**In NEST 3**
```bash
export PYNEST_QUIET=1
```
```python
import nest
```

@jessica-mitchell: Can you please add this information to the 2→3 conversion guide? Thanks

File: doc/guides/using_nest_with_music.rst [hunk 2]
```diff
@@ -69,8 +69,8 @@ continuing.
 
    import nest
 
-   nest.sli_run("statusdict/have_music ::")
-   if not nest.spp():
+   nest.ll_api.sli_run("statusdict/have_music ::")
+   if not nest.ll_api.spp():
```
[the remark this answers is not in the document]
Comment (steffengraber) [reply doc/guides/using_nest_with_music.rst]: Done

File: doc/guides/using_nest_with_music.rst [hunk 3]
```diff
@@ -124,8 +124,8 @@ script, but without the spike generator.
 
   import nest
 
-  nest.sli_run("statusdict/have_music ::")
-  if not nest.spp():
+  nest.ll_api.sli_run("statusdict/have_music ::")
+  if not nest.ll_api.spp():
```
[the remark this answers is not in the document]
Comment (steffengraber) [reply doc/guides/using_nest_with_music.rst]: Done

**Comment by steffengraber** [comment]
I shortened the output and add it to the documentation.

**Pull request merged**